src/views/catalog/CreateFromInstanceTypes/state/utils/state.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { InstanceTypeVMState, InstanceTypeVMStoreState } from './types';
 
 export const instanceTypeVMInitialState: InstanceTypeVMState = {
+  customDiskSize: '',
```

Comment:
shouldn't this have a default size 30Gi like in template flow?

Reply:
I want to ensure the disk size is empty when the user does not customize it. With that, I'll add the `spec.storage.resources.requests.storage` details only when the user customize the size.